src/Resources/Resources/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 1.7.1
+
+* Update dependency assemlby Microsoft.Extensions.Caching.Memory from 1.1.1 to 2.2
```

Comment:
```suggestion
* Update dependency assembly Microsoft.Extensions.Caching.Memory from 1.1.1 to 2.2
```

Reply:
Note - thsi is a minor type, I would not hold up the release for this, but if we merge any further PRs, we could fix this